### PR TITLE
Adding overdue_items report query

### LINKED ISSUE #453: RA/UM - build overdue items report
https://issues.folio.org/browse/UXPROD-2399

### PR DESCRIPTION
**Pull request by amzoss**
Should work on any standard day of folio_snapshot. Fixes #453

### CODE REVIEW

**Review by adoerrer — CHANGES_REQUESTED** [review]
Beside the minor requested change it works like a charm, thanks!

File: sql/report_queries/overdue_items/overdue_items.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+WITH parameters AS (
+    SELECT
+        /* replace the placeholder number with the number of days overdue that is needed for this report */
+        '2000'::integer AS days_overdue_filter, -- doesn't work if empty
+        /* Fill in a location name, OR leave blank for all locations */
+        ''::varchar AS current_item_permanent_location_filter, --Online, Annex, Main Library
+        ''::varchar AS current_item_temporary_location_filter, --Online, Annex, Main Library
+        ''::varchar AS current_item_effective_location_filter, --Online, Annex, Main Library 
+        /*Choose a patron group name or leave blank*/
+        ''AS patron_group_filter --can use for Borrow Direct or ILL
```
Comment (jenmawe) [inline sql/report_queries/overdue_items/overdue_items.sql:22]: I tried this with ::varchar AS and it worked.

**Review by jenmawe — CHANGES_REQUESTED** [review]

File: sql/report_queries/overdue_items/overdue_items.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (amzoss) [reply sql/report_queries/overdue_items/overdue_items.sql]: Thanks so much! I think this is fixed now.

Comment (amzoss) [reply sql/report_queries/overdue_items/overdue_items.sql]: Thanks so much! I think this is fixed now.

**Review by adoerrer — APPROVED** [review]
Great, thanks!

**Review by jenmawe — APPROVED** [review]

**Pull request merged**